mozjs stuff is actually in /unstable not in /stable

Causes bugs like http://bugs.gentoo.org/275318

Upstream bug: https://bugzilla.mozilla.org/show_bug.cgi?id=500645

Will likely be fixed in 1.9.2

---
--- xulrunner/installer/mozilla-js.pc.in	2009-06-26 08:37:34.458635913 -0500
+++ xulrunner/installer/mozilla-js.pc.in	2009-06-26 08:38:13.623656097 -0500
@@ -1,10 +1,11 @@
 prefix=%prefix%
 sdkdir=%sdkdir%
 includedir=%includedir%
+includetypes=unstable
 
 Name: JavaScript
 Description: The Mozilla JavaScript Library
 Version: %MOZILLA_VERSION%
 Requires: %NSPR_NAME% >= %NSPR_VERSION%
 Libs: -L${sdkdir}/lib -lmozjs
-Cflags: -I${includedir}/stable -DXP_UNIX -DJS_THREADSAFE
+Cflags: -I${includedir}/${includetypes} -DXP_UNIX -DJS_THREADSAFE
